lua/lualine/init.lua

```diff
@@ -239,33 +132,34 @@ local function tabline() return statusline(config.tabline, true) end
 
 local function setup_theme()
   local async_loader
-  async_loader = vim.loop.new_async(vim.schedule_wrap(function()
-    local function get_theme_from_config()
-      local theme_name = config.options.theme
-      if type(theme_name) == 'string' then
-        local ok, theme = pcall(require, 'lualine.themes.' .. theme_name)
-        if ok then return theme end
-      elseif type(theme_name) == 'table' then
-        -- use the provided theme as-is
-        return config.options.theme
-      end
-      vim.api.nvim_echo({
-        {
-          'theme ' .. tostring(theme_name) .. ' not found, defaulting to gruvbox',
-          'ErrorMsg'
-        }
-      }, true, {})
-      return require 'lualine.themes.gruvbox'
-    end
-    local theme = get_theme_from_config()
-    highlight.create_highlight_groups(theme)
-    vim.api.nvim_exec([[
+  async_loader = vim.loop.new_async(vim.schedule_wrap(
+                                        function()
+        local function get_theme_from_config()
+          local theme_name = config.options.theme
+          if type(theme_name) == 'string' then
+            local ok, theme = pcall(require, 'lualine.themes.' .. theme_name)
+            if ok then return theme end
+          elseif type(theme_name) == 'table' then
+            -- use the provided theme as-is
+            return config.options.theme
+          end
+          vim.api.nvim_echo({
```

Comment:
I we should use `nvim_err_writeln()` instead of `nvim_echo`.
Or maybe `nvim_notify` . What do you think ?

Reply:
I will check both of these functions out and let you know what I think

Reply:
`nvim_err_writeln()` has simpler semantics and `nvim_notify()` can be redirected to where ever user likes . For example mine goes to a float instead of messages .

Reply:
I've went with `nvim_err_writeln` since it does exactly what we need it to do with no maintanence needed on our side :smile: